csfieldguide/chapters/content/en/human-computer-interaction/sections/interface-usability.md

```diff
@@ -109,6 +109,8 @@ If a system takes some time to respond (say, 10 seconds) then chances are the us
 For example, if you have a phone number to type in that someone has just told you, and it takes 12 seconds before you can type it, you may forget the number, whereas if you can access the interface within a couple of seconds, you can probably enter the number without much effort.
 For this reason, any part of a system that takes more than about 10 seconds to respond forces the user to rehearse or write down key information, which is more tiring.
 
+{interactive slug="number-memory" type="in-page"}
```

Comment:
Maybe add some sort of text to introduce the interactive (don't have any suggestions right now).

Reply:
The text immediately above is perfect for it, so I don't need to add anything

Reply:
I don't know if it will still be perfect when my PR is merged in.

Reply:
The important part is this sentence:

> For example, if you have a phone number to type in that someone has just told you, and it takes 12 seconds before you can type it, you may forget the number, whereas if you can access the interface within a couple of seconds, you can probably enter the number without much effort.

This comes after your interactive so it still works